Modules/System/Digipoort/src/app.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id":  "93f93dda-3177-4de6-9f6e-b38103809c6d",
+    "name":  "Tax Management",
+    "publisher":  "Microsoft",
+    "brief":  "Provides tax helper functions",
+    "description":  "Provides functions to submit tax and receive response messages",
+    "version":  "16.0.0.0",
+    "privacyStatement":  "https://go.microsoft.com/fwlink/?linkid=724009",
+    "EULA":  "https://go.microsoft.com/fwlink/?linkid=2009120",
+    "help":  "https://go.microsoft.com/fwlink/?linkid=2103698",
+    "url":  "https://go.microsoft.com/fwlink/?linkid=724011",
+    "logo":  "",
+    "dependencies":  [
+                  {"appId": "437dbf0e-84ff-417a-965d-ed2bb9650972",
```

Comment:
This dependency is not possible. The System Application cannot reference the Base Application.